src/components/Combobox/Combobox.tsx

```diff
@@ -173,7 +173,7 @@ export const Combobox: React.FC<ComboboxProps> = ({
       }
       setOpenMenu(false);
     }
-    useAriaLive('');
+    setAriaLiveText('');
```

Comment:
We might want to reset with `null` instead of an empty string. That makes it a little clearer that the text is unset.

Reply:
Yess